packages/ui-pattern-block/src/components/ResponsivePreview.tsx

```diff
@@ -0,0 +1,106 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
Can't change the preview device by keyboard only

Reply:
Fixed